pkg/controller/launcher/job.go

```diff
@@ -239,6 +243,41 @@ func getBatchJob(clusterName string, imageURI string, curator clustercuratorv1.C
 				},
 			},
 		}
+	case "installPosthook", "upgradePosthook":
+		if (desiredCuration == "installPosthook" && curator.Spec.Install.Posthook != nil) || (desiredCuration == "upgradePosthook" && curator.Spec.Upgrade.Posthook != nil) {
+			isPosthook = true
+		}
+		newJob = &batchv1.Job{
+			ObjectMeta: v1.ObjectMeta{
+				GenerateName: "curator-job-",
+				Namespace:    clusterName,
+				Labels: map[string]string{
+					"open-cluster-management": "curator-job",
+				},
+				Annotations: map[string]string{
+					PostAJob:     "Retry an install posthook job",
```

Comment:
```suggestion
					PostAJob:     "Retry a posthook job",
```

or figure out if it's install or upgrade.

Reply:
Thanks Mike. Good catch! Fixed.